Makefile

```diff
@@ -136,7 +136,7 @@ verify-fast: verify-go verify-assets verify-sh verify-py verify-config verify-rf
 
 # Full verification checks that should run in CI
 .PHONY: verify-ci
-verify-ci: verify-fast verify-images verify-licenses verify-containers
+verify-ci: verify-fast verify-images verify-containers
```

Comment:
Why was the license verification removed?

Reply:
The commit message for that commit shows the error message that is breaking the job.

```
2023/07/08 18:18:45 failed to evaluate licenses: invalid character 'G' looking for beginning of value
```

It affects all PRs, so I suspect an updated version of the tool is behaving differently. We don't add new dependencies often, so I thought rather than spending time figuring out why that tool is reporting an error I would just turn it off.

Reply:
See https://github.com/openshift/microshift/pull/1987#issuecomment-1627712945 for example

Reply:
Interesting. This problem does not occur on my development machine.

Reply:
I was not able to reproduce it locally either. Perhaps it has something to do with the compiler version? Either way, I wanted to unblock us without spending time on a low-value tool.

Reply:
I implemented a w/a in the https://github.com/openshift/microshift/pull/2020 PR